README.md

```diff
@@ -159,3 +159,107 @@ spec:
         - "--dns01-recursive-nameservers=1.1.1.1:53"
         - "--dns01-recursive-nameservers-only"
 ```
+## Enabling metrics and monitoring for `cert-manager`
+
+Cert-Manager exposes controller metrics in the format expected by [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator). 
+
+We will be using the `ServiceMonitor` object in the operand's namespace (by default `cert-manager`), make sure that your instance of the Prometheus Operator is properly configured to find it.
+
+You can check `.spec.serviceMonitorNamespaceSelector` and `.spec.serviceMonitorSelector` fields of `prometheus` resource.
```

Comment:
I think, it'd be more helpful to have mention the expected values in these fields. Rephrase " `.spec.serviceMonitorNamespaceSelector` and `.spec.serviceMonitorSelector` fields of `prometheus` resource should contain corresponding  <values>."